### DISCUSSION TITLE
Galaxy Fund's Support for BuckyOS Demo Contributors!

### DISCUSSION THREAD

**Discussion by glaxyfund** [buckyos/buckyos#55] [Announcements]
Dear Contributors,

I am pleased to announce that a few of us at Galaxy Fund, who have been long-time NAS users, recently had the opportunity to experience the BuckyOS Demo. We believe that the demo has already demonstrated some of the key design principles and holds the potential to be a true game-changer in the private storage cloud space.

In light of this, we have decided to extend a small support fund of 31,500 USDT in accordance with the rules of the BuckyOS DAO. With @waterflier  assistance, we’ve created an investment contract that allows all contributors to the demo version to redeem their share of this fund.

The exchange rate is set at 1 USDT : 100 BDT, and all contributors will receive an equal proportion of BDT tokens based on this rate.

(https://dao.buckyos.org/invest/2)

Thank you for your hard work and dedication to the BuckyOS project. We look forward to seeing BuckyOS grow and succeed, and we're excited to be a part of this journey with you!

**Comment by waterflier** [comment]
Thank you for your generous support and for believing in the potential of BuckyOS! 

We are incredibly grateful for your investment, especially given how complex the current DAO investment page can be. Your contribution is a significant boost to our project, and it encourages all of us to keep pushing forward.

We’re excited to continue improving BuckyOS, and we can’t wait to share the next version with everyone. Stay tuned—there’s much more to come!

Thanks again for your support!
